BreanaMarie/w03-d02homework

```diff
@@ -0,0 +1,7 @@
+GET ALL ICECREAM "GET /icecream"
+GET ICECREAM BY FLAVOR "GET /icecream/:flavors"
+GET ALL ICECREAM FLAVORS "GET /icecream/flavors"
+GET FLAVOR BY FRUITY "GET /fruity/:flavor"
+GET FLAVOR BY GLUTEN FREE "GET /gluten_free/:flavor"
+GET FLAVOR BY CHUNKY "GET /chunky/:flavor"
+PUT NEW FLAVOR "PUT/icecream/:flavor"
```

Comment:
Interesting use of PUT!  We usually use POST for that, but it could be PUT.  
This related article might be interesting: http://stackoverflow.com/questions/630453/put-vs-post-in-rest